services/docker-compose.local.yml

```diff
@@ -91,7 +91,7 @@ services:
   notifications:
     environment:
       <<: *common_environment
-      INVITATIONS_REMOTE_DEBUGGING_PORT : 3000
+      NOTIFICATIONS_REMOTE_DEBUGGING_PORT : 3000
```

Comment:
@giancarloromeo this is the fix to run it locally

Reply:
Many Thanks